Automattic/sensei
# Add course enrollment indicator to learner management

_Built off of #2799_

This adds a simple indicator to the learner management student listing screens (currently on courses _and_ lessons, but we could limit this just to the courses view). Right now, hovering over the status will show which providers are handling enrollment for the course and of those, which (if any) are enrolling the student. 

In its current state, this will be helpful during development but also in production to track down where enrollment is/isn't coming from. Further down, it would be nice to add additional information and/or ways to discover, for example, which order/membership is giving access.

<img width="1650" alt="Screen Shot 2020-01-13 at 9 23 19 PM" src="https://user-images.githubusercontent.com/68693/72293260-0e626d00-364b-11ea-9baf-0783ddb52258.png">


### Testing Instructions
- Add the code snippet `add_filter( 'sensei_feature_flag_enrolment_provider_tooltip', '__return_true' );` to enable tooltips.
- From one of the WCPC WIPs, sign up for a course using one of the available access providers (such as buying a simple product associated with the course).
- Manage students for the course in Learner Management.
- Hover over Enrollment status for an individual to see which provider is giving access.
- If no providers are handling enrollment for a course, you'll see a message indicating so when you hover over. We could spruce this up before it hits production.

### New Hook
- `sensei_feature_flag_enrolment_provider_tooltip` - Enable enrollment tooltips in Learner Management. (Probably not worth mentioning in the change log since it's meant to be a transitory filter.)
